colosseum/validators.py

```diff
@@ -0,0 +1,101 @@
+from . import parser
+from . import units
+
+
+def _numeric_validator(value, numeric_type, min_value, max_value):
+    error_msg = ''
+    try:
+        value = numeric_type(value)
+    except (ValueError, TypeError):
+        error_msg = "Cannot coerce {value} to {numeric_type}".format(
+            value=value, numeric_type=numeric_type.__name__)
+
+    if not error_msg and min_value is not None and value < min_value:
+        error_msg = 'Value {value} bellow minimum value {min_value}'.format(
+            value, min_value)
+
+    if not error_msg and max_value is not None and value > max_value:
+        error_msg = 'Value {value} above maximum value {max_value}'.format(
+            value, max_value)
+
+    return error_msg, value
+
+
+def is_number(value, min_value=None, max_value=None):
+    """
+    Validate that value is a valid float.
+
+    If min_value or max_value are provided, range checks are performed.
+    """
+
+    def validator(value):
+        return _numeric_validator(value, numeric_type=float, min_value=min_value, max_value=max_value)
```

Comment:
To avoid confusion (and potential error), it would be better to use a different variable name for `value` internally. It's entirely *legal* to re-use the variable name, but from a 'human reading the code' perspective, it's may be unclear which value is being used (since, in theory, they could be different)